modules/react-mapbox/src/types/internal.ts

```diff
@@ -1,34 +1,15 @@
 // Internal types
-import type {
-  Map,
+import type {Map} from 'mapbox-gl';
+
+export type Transform = Map['transform'];
+
+export type {
   GeoJSONSource as GeoJSONSourceImplementation,
-  ImageSource as ImageSourceImplemtation,
-  CanvasSource as CanvasSourceImplemtation,
+  ImageSource as ImageSourceImplementation,
+  CanvasSource as CanvasSourceImplementation,
```

Comment:
And then are these "Implementation" types the official name? 

Are they private? (I don't see them mentioned in the upgrade guide)

Reply:
Oh, I see these are in the internal type file